automatic/sublimetext4/tools/chocolateyUninstall.ps1

```diff
@@ -1,26 +1,29 @@
-$ErrorActionPreference = 'Stop'
+﻿$ErrorActionPreference = 'Stop'
 
-$packageName = "sublimetext4"
+$packageName = $env:ChocolateyPackageName
+$softwareName = 'Sublime Text'
 
-[array]$key = Get-UninstallRegistryKey -SoftwareName "Sublime Text 4"
+[array]$key = Get-UninstallRegistryKey -SoftwareName $softwareName
 
 if ($key.Count -eq 1) {
   $key | ForEach-Object {
     $packageArgs = @{
-      packageName = $packageName
-      fileType    = 'exe'
-      silentArgs  = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /TASKS="contextentry"'
-      validExitCodes= @(0)
-      file          = "$($_.UninstallString.Trim('"'))"
+      packageName    = $packageName
+      fileType       = 'exe'
+      silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /TASKS="contextentry"'
+      validExitCodes = @(0)
+      file           = "$($_.UninstallString.Trim('"'))"
+      softwareName   = $softwareName
     }
- 
+    Uninstall-BinFile -Name 'subl'
```

Comment:
I think we should always remove the shim, regardless of whether we manage to find a single registry key or not.

Reply:
64f928cf384e41dc71ebdedfa9162fa1333a62cb should take care of that.